packages/realm-server/tests/helpers/index.ts

```diff
@@ -199,6 +201,7 @@ export async function runBaseRealmServer(
   virtualNetwork: VirtualNetwork,
   queue: Queue,
   dbAdapter: PgAdapter,
+  permissions: RealmPermissions = { '*': ['read'] },
```

Comment:
we should probably default to whatever the default realm permissions are--i think only the base realm defaults to only world readable